Fix typos and grammar errors across documentation (#144709)
python/cpython: Doc/library/tracemalloc.rst

   When a snapshot is taken, tracebacks of traces are limited to
   :func:`get_traceback_limit` frames. See the :func:`take_snapshot` function.
   The original number of frames of the traceback is stored in the
   :attr:`Traceback.total_nframe` attribute. That allows one to know if a traceback
   has been truncated by the traceback limit.

>    The :attr:`Trace.traceback` attribute is a class:`Traceback` instance.

Reviewer: sorry a typo in my previous suggestion

```suggestion
   The :attr:`Trace.traceback` attribute is a :class:`Traceback` instance.
```

>    The :attr:`Trace.traceback` attribute is an instance of :class:`Traceback`.

Reviewer: ```suggestion
   The :attr:`Trace.traceback` attribute is a class:`Traceback` instance.
```
